silo/tasks.py

```diff
@@ -0,0 +1,57 @@
+# Create your tasks here
+from __future__ import absolute_import, unicode_literals
+from celery import shared_task
+from celery.result import AsyncResult
+from tola.celery import app
+
+from random import randint
+from silo.custom_csv_dict_reader import CustomDictReader
+from tola.util import importJSON, saveDataToSilo
+from .models import Silo, Read
+
+import logging
+
+logger = logging.getLogger("tola")
+
+#@shared_task(bind=True, max_retries=3)
+@app.task(bind=True, max_retries=3)
+def async_rand(self, nr):
+    n = randint(0, 100)
+    print("Celery Task Called")
```

Comment:
If you wanna log when a task is called, you should use the `info` method of logging instead.